test/helpers/run-browser.js

```diff
@@ -20,6 +20,7 @@ function runBrowser(config) {
     puppeteer
       .launch({
         headless: true,
+        ignoreHTTPSErrors: true,
```

Comment:
Due invalid localhost cert? Let's add comment about it

Reply:
yes, done 👍🏻